docs/loading/Spark-connector-starrocks.md

```diff
@@ -348,3 +378,335 @@ The following example explains how to load data with Spark SQL by using the `INS
     +------+-----------+-------+
     2 rows in set (0.00 sec)
     ```
+
+## Best Practices
+
+### Load data to Primary Key table
+
+This section will show how to load data to StarRocks Primary Key table to achieve partial updates, and conditional updates.
+You can see [Change data through loading](../loading/Load_to_Primary_Key_tables) for the detailed introduction of these features.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/f7d77cf2-b583-4ca7-9c84-38d62c7408ff)
-> For more information, see XXX.